Tests/PARStoreTests.m

```diff
@@ -190,6 +190,32 @@ - (void)testPropertyListSetter
     document2 = nil;
 }
 
+- (void)testNilSetter
+{
+    NSString *title = @"Some title";
+    NSString *first = @"Albert";
+    
+    // first load = create document and save data
+    NSURL *url = [[self urlWithUniqueTmpDirectory] URLByAppendingPathComponent:@"doc.parstore"];
+    PARStoreExample *document1 = [PARStoreExample storeWithURL:url deviceIdentifier:[self deviceIdentifierForTest]];
+    [document1 loadNow];
+    document1.title = title;
+    document1.first = first;
+    document1.first = nil;
+    XCTAssertEqualObjects(document1.title, title, @"unexpected 'title' value : '%@' instead of '%@'", document1.title, title);
+    XCTAssertNil(document1.first, @"unexpected 'first' value: '%@' instead of nil", document1.first);
+    [document1 tearDownNow];
+    document1 = nil;
+    
+    // second load = load document and compare data
+    PARStoreExample *document2 = [PARStoreExample storeWithURL:url deviceIdentifier:[self deviceIdentifierForTest]];
+    [document2 loadNow];
+    XCTAssertEqualObjects(document2.title, title, @"unexpected 'title' value : '%@' instead of '%@'", document2.title, title);
+    XCTAssertNil(document2.first, @"unexpected 'first' value: '%@' instead of nil", document2.first);
+    [document2 tearDownNow];
+    document2 = nil;
+}
```

Comment:
Not really that important, but the `loadNow` and `tearDownNow` may be pointing to these tests being a bit too broad. Usually, a single standard `setup` and `teardown` should be enough for a single test. So instead of `testNilSetter`, you might have `testSavingAndLoadingDataWithNil` and `testLoadingDataWithNilFromSecondDocument`.

I wouldn't change anything now, but maybe worth thinking about for future tests.